src/sass/mixins/_Animation.scss

```diff
@@ -0,0 +1,345 @@
+}
+
+// Slide
+@mixin ms-u-slideRightIn10 {
+  @include ms-animation((fadeIn, slideRightIn10), $ms-duration3, $ms-ease1);
+}
+
+@mixin ms-u-slideRightIn20 {
+  @include ms-animation((fadeIn, slideRightIn20), $ms-duration3, $ms-ease1);
+}
+
+@mixin ms-u-slideRightIn40 {
+  @include ms-animation((fadeIn, slideRightIn40), $ms-duration3, $ms-ease1);
+}
+
+@mixin ms-u-slideRightIn400 {
+  @include ms-animation((fadeIn, slideRightIn400), $ms-duration3, $ms-ease1);
+}
+
+@mixin ms-u-slideRightOut40 {
+  @include ms-animation((fadeOut, slideRightOut40), $ms-duration1, $ms-ease2);
+}
+
+@mixin ms-u-slideRightOut400 {
+  @include ms-animation((fadeOut, slideRightOut400), $ms-duration1, $ms-ease2);
+}
+
+@mixin ms-u-slideLeftIn10 {
+  @include ms-animation((fadeIn, slideLeftIn10), $ms-duration3, $ms-ease1);
+}
+
+@mixin ms-u-slideLeftIn20 {
+  @include ms-animation((fadeIn, slideLeftIn20), $ms-duration3, $ms-ease1);
+}
+
+@mixin ms-u-slideLeftIn40 {
+  @include ms-animation((fadeIn, slideLeftIn40), $ms-duration3, $ms-ease1);
+}
+
+@mixin ms-u-slideLeftIn400 {
+  @include ms-animation((fadeIn, slideLeftIn400), $ms-duration3, $ms-ease1);
+}
+
+@mixin ms-u-slideLeftOut40 {
+  @include ms-animation((fadeOut, slideLeftOut40), $ms-duration1, $ms-ease2);
+}
+
+@mixin ms-u-slideLeftOut400 {
+  @include ms-animation((fadeOut, slideLeftOut400), $ms-duration1, $ms-ease2);
+}
+
+@mixin ms-u-slideUpIn10 {
+  @include ms-animation((fadeIn, slideUpIn10), $ms-duration1, $ms-ease2);
+}
+
+@mixin ms-u-slideUpIn20 {
+  @include ms-animation((fadeIn, slideUpIn20), $ms-duration3, $ms-ease1);
+}
+
+@mixin ms-u-slideDownIn10 {
+  @include ms-animation((fadeIn, slideDownIn10), $ms-duration1, $ms-ease2);
+}
+
+@mixin ms-u-slideDownIn20 {
+  @include ms-animation((fadeIn, slideDownIn20), $ms-duration3, $ms-ease1);
+}
+
+@mixin ms-u-slideUpOut10 {
+  @include ms-animation((fadeOut, slideUpOut10), $ms-duration1, $ms-ease2);
+}
+
+@mixin ms-u-slideUpOut20 {
+  @include ms-animation((fadeOut, slideUpOut20), $ms-duration1, $ms-ease2);
+}
+
+@mixin ms-u-slideDownOut10 {
+  @include ms-animation((fadeOut, slideDownOut10), $ms-duration1, $ms-ease2);
+}
+
+@mixin ms-u-slideDownOut20 {
+  @include ms-animation((fadeOut, slideDownOut20), $ms-duration1, $ms-ease2);
+}
+
+// Scale
+@mixin ms-u-scaleUpIn100 {
+  @include ms-animation((fadeIn, scaleUp100), $ms-duration3, $ms-ease1);
+}
+
+@mixin ms-u-scaleUpOut103 {
+  @include ms-animation((fadeOut, scaleUp103), $ms-duration1, $ms-ease2);
+}
+
+@mixin ms-u-scaleDownOut98 {
+  @include ms-animation((fadeOut, scaleDown98), $ms-duration1, $ms-ease2);
+}
+
+@mixin ms-u-scaleDownIn100 {
+  @include ms-animation((fadeIn, scaleDown100), $ms-duration3, $ms-ease1);
+}
+
+// Rotate
+@mixin ms-u-rotate90deg {
+  @include ms-animation(rotate90, $ms-duration1, $ms-ease2);
+}
+
+@mixin ms-u-rotateN90deg {
+  @include ms-animation(rotateN90, $ms-duration1, $ms-ease2);
+}
+
+
+// Fade
+@mixin ms-u-fadeIn100 {
+  animation-duration: 0.1s;
```

Comment:
Totally understand the motivation here and agree that the original name/value mismatch for these was confusing, but I'm quite sure at the time the values were intentional/by design. While it's probably fine to change them to make more sense, please check with motion design to verify. 

Frankly, I'd prefer we use simple, predictable durations in sensible increments like you've proposed here.